README.md

```diff
@@ -1,6 +1,11 @@
 tried: \>>>> [this](https://twitter.com/spacerezt/status/903047646549561344) <<<<
 
 todo: 
-- where to get images? 
-- improve spacing / centering of divs and imgs
+- (kind of) where to get images? 
+
+todid:
+- improve spacing / centering of divs and imgs (make 100% height & width)
```

Comment:
Oh ok nice! Now they appear in the center? I need to get this to work so I can actually take a look 😅